modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -269,6 +276,9 @@ public Response wipeData(@HeaderParam("Accept") String acceptHeader,
 		} catch (DeviceManagementException e) {
 			message.setResponseMessage("Issue in retrieving device management service instance");
 			throw new AndroidOperationException(message, responseMediaType);
+		} catch (OperationConfigurationException e) {
+			message.setResponseMessage("Issue in setting up payload in operation");
+			throw new AndroidOperationException(message, responseMediaType);
```

Comment:
It should be better to log the errors here rather than just throwing (To facilitate better debugging)